.github/workflows/build-docker-image-to-prod.yml

```diff
@@ -90,9 +94,13 @@ jobs:
           token: ${{ secrets.DEPOT_PROJECT_TOKEN }}
           project: 64mmf0n610
           context: frontend
-          tags: infisical/frontend:test
+          tags: |
+            infisical/frontend:${{ steps.extract_version.outputs.version }}
```

Comment:
this is for creating a test image before publishing the real thing. you can revert this